doc/index.rst

```diff
@@ -196,7 +196,6 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    :hidden:
 
    code/qml
-   code/qml_allocation
```

Comment:
@albi3ro  I removed this from the TOC because there's really not a strong use for it at the moment :)